erpnext/selling/doctype/customer/customer.py

```diff
@@ -136,7 +136,7 @@ def update_lead_status(self):
 		if self.lead_name:
 			lead = frappe.get_doc('Lead', self.lead_name)
 			lead.status = 'Converted'
-			lead.save()
+			lead.save(ignore_permissions=True)
```

Comment:
instead of getting doc and saving. You can use frappe.db.set_value

Reply:
Done